gradle/pom-aar.gradle

```diff
@@ -0,0 +1,45 @@
+publishing.publications.all {
+    //TODO github profile?
+    def devs = ['szczepiq:Szczepan Faber', 'bric3:Brice Dutheil', 'raphw:Rafael Winterhalter', 'TimvdLippe:Tim van der Lippe']
```

Comment:
This is a lot of duplicate code. Can we reuse by first running the java-library pom.xml and then append the version of Mockito afterwards?